samples/bluetooth/nrf_dm/README.rst

```diff
@@ -78,6 +78,30 @@ The addresses of these devices are stored in a list.
 When a device does not respond within a certain period of time, it is removed from the list.
 Also the last measurement results with the peers are stored in this list.
 
+Energy consumption and the number of measurements performed in a given time depend on:
+
+   * Advertiser settings.
+   * Scanner settings.
+   * Extended size of measurement window by ``extra_window_time_us`` in the :c:struct:`dm_request` structure.
+
+The shorter the interval, the higher the frequency of broadcasting advertising packages.
+This leads to a higher probability of synchronization and to a higher number of measurements taken at a given time.
+However, the higher number of packets transmitted also results in higher power consumption.
+
+The scan interval and scan window parameters determine how often and how long the scanning device will listen for packets.
+Like the short advertising interval, these values also have an impact on energy consumption.
+
+How often the measurement will be executed is also managed by :kconfig:option:`CONFIG_DM_MIN_TIME_BETWEEN_TIMESLOTS_US`.
+The :kconfig:option:`CONFIG_DM_TIMESLOT_QUEUE_COUNT_SAME_PEER` and :kconfig:option:`CONFIG_DM_TIMESLOT_QUEUE_LENGTH` Kconfig options also indirectly affect this.
+
+Extending the measurement window will increase the probability of measurement success for coarse synchronization.
+However, it affects the radio's turn-on time and thus, power consumption.
+
+At the time of application runtime, the additional delay of measurement execution is configured using ``start_delay_us`` in the :c:struct:`dm_request` structure.
+This option helps adjust the synchronization.
+:kconfig:option:`CONFIG_DM_INITIATOR_DELAY_US` Kconfig options defines the initiator delay.
+Increasing these parameters increases the latency.
```

Comment:
```suggestion
This option helps in adjusting the synchronization.
The :kconfig:option:`CONFIG_DM_INITIATOR_DELAY_US` Kconfig option defines the initiator delay.
Increasing these parameters increases the latency.
```
Are the parameters increased or their values?

Reply:
It is about the value of these parameters.